Automattic/WP-Job-Manager
# Fix company logo extending outside parent

Fixes #

#### Changes proposed in this Pull Request:

* Fix company logo extending outside parent `div` on single job page.

#### Testing instructions:

* Add a job listing with a company logo.
* Using the Storefront or Twenty Seventeen themes, view the job listing on the front-end.
* Ensure the company logo thumbnail does not extend outside of its parent.
* Repeat the above with other themes.


#### Proposed changelog entry for your changes:
Fix company logo extending outside of parent for some themes
